data/Viral kinetics of SARS-CoV-2/data.yaml

```diff
@@ -0,0 +1,105 @@
+analytes:
+  throatswab_SARSCoV2:
+    biomarker: SARS-CoV-2
+    description: SARS-CoV-2 RNA genome copy concentration in throat swab samples.
+      Specimens were collected from all patients at least 2 days after hospitalization
+      and physicians checked their symptoms and signs daily.
+    gene_target: unknown
+    limit_of_detection: unknown
+    limit_of_quantification: 1000
+    reference_event: PatientID
+    specimen: throat_swab
+    unit: value
+description: The authors measured SARS-CoV-2 in longitudinal throat swab samples collected
+  from 71 COVID-19 patients between February 4 and April 7, 2020. Abundances were
+  quantified using diagnosed by real-time reverse transcription polymerase chain reaction
+  (RT-PCR). Specimens were collected from all patients at least 2 days after hospitalization
+  and physicians checked their symptoms and signs daily. Patients who had asymptomatic
+  carrier and incubation period were analyzed.
+doi: 10.1016/j.ijid.2020.04.083
+participants:
+- attributes:
```

Comment:
For the `attributes`, please keep `age`, `sex`, and `death` (currently you are using `died`). We need to include `death` in the schema.